include/deal.II/grid/grid_tools.h

```diff
@@ -2269,6 +2269,35 @@ namespace GridTools
   void copy_boundary_to_manifold_id(Triangulation<dim, spacedim> &tria,
                                     const bool reset_boundary_ids=false);
 
+  /**
+   * Map the given boundary ids to the given manifold ids on faces and
+   * edges at the boundary.
+   *
+   * This function copies the boundary ids of the boundary faces and
+   * edges that are present in the parameter @p src_boundary_ids to
+   * the corresponding manifold_id in @p dst_manifold_ids, of the same
```

Comment:
corresponding manifold_id in @p dst_manifold_ids
->
corresponding manifold id in @p dst_manifold_ids